test/infrastructure/docker/config/manager/manager.yaml

```diff
@@ -22,6 +22,19 @@ spec:
         - "--feature-gates=MachinePool=${EXP_MACHINE_POOL:=false},ClusterTopology=${CLUSTER_TOPOLOGY:=false}"
         image: controller:latest
         name: manager
+        env:
```

Comment:
why do we need the env variables for CAPD? Is this expected of other providers too?

Reply:
Technically it's not required at all.
If a controller uses the ClusterCacheTracker **and** if the controller wants to access the cluster it's running on directly then the env vars are required.